prometheus/base/prometheus.rules.yaml

```diff
@@ -11,3 +11,9 @@ spec:
       rules:
         - record: o11y_prometheus_http_requests_total
           expr: sum by (code) (prometheus_http_requests_total)
+        - record: container:network_transmit_bytes:group
```

Comment:
Are these changes related to the extraction of the Prometheus rules ?

Reply:
Yes. They make use of a rule. It proves that the extraction works.